starter-code/src/viking.js

```diff
@@ -1,11 +1,126 @@
 // Soldier
-function Soldier () {}
+function Soldier (health, strength) {
+  this.health = health;
+  this.strength = strength;
+}
+
+Soldier.prototype.attack = function() {
+  return this.strength;
+}
+
+Soldier.prototype.receiveDamage = function(damage) {
+  this.health= this.health - damage;
+}
 
 // Viking
-function Viking () {}
+function Viking (name, health, strength) {
+  this.name = name;
+  Soldier.call(this, health, strength);
+  War.call(this);
+
+
+}
 
+Viking.prototype = Object.create(Soldier.prototype);
+Viking.prototype.constructor = Viking;
+
+Viking.prototype.receiveDamage = function (damage){
+    if (this.health - damage > 0 ) {
+      this.health = this.health - damage;
+      return this.name + " has received "+damage+" points of damage";
+
+    }else {
+      this.health = this.health - damage;
+      return this.name + " has died in act of combat";
+    }
+}
+Viking.prototype.battleCry = function (){
+  return "Odin Owns You All!";
+}
 // Saxon
-function Saxon () {}
+function Saxon (health, strength) {
+  Soldier.call(this, health, strength);
+  War.call(this);
+}
```

Comment:
La llamada a las propiedades de War, `War.call(this)` no tiene sentido. Sólo la llamada a las propiedades del padre, `Soldier.call(this, health, strength)`